raster_download/src/main/java/de/meggsimum/sauber/sdi/RasterDownloader.java

```diff
@@ -157,6 +162,33 @@ public void resolveSecrets() throws Exception {
 			} else {
 				throw new FileNotFoundException("Not able not load HHI IP address from secrets");
 			}
+			
+			File secretFileGeoserverUserCtn = new File("/run/secrets/geoserver_user");
+			File secretFileGeoserverUserLoc = new File(workingDir + "/../secrets/geoserver_user.txt");
+			if (secretFileGeoserverUserCtn.exists()) {
+				InputStream fis = new FileInputStream(secretFileGeoserverUserCtn);
+				this.geoserverUser = IOUtils.toString(fis, "UTF-8");
+			} else if (secretFileGeoserverUserLoc.exists()) {
+				System.out.println("Using local backup secret at " + secretFileGeoserverUserLoc.getAbsolutePath());
+				InputStream fis = new FileInputStream(secretFileGeoserverUserLoc);
+				this.geoserverUser = IOUtils.toString(fis, "UTF-8");
+			} else {
+				throw new FileNotFoundException("Not able not load GeoServer user from secrets");
```

Comment:
```suggestion
				throw new FileNotFoundException("Not able to load GeoServer user from secrets");
```

Reply:
this typo seems to be in other parts of the code as well. like [here](https://github.com/meggsimum/sauber-sdi-docker/pull/157/files#diff-9a079b4f485f1074d265e581ee50ff7f01261ad70dd0a9a6083f7cb2eb18c7e3R163)